### PR TITLE
Text lint bear2048

### LINKED ISSUE #2048: TextLintBearTest: Close opened test files properly
py.test gives warnings on https://circleci.com/gh/coala/coala-bears/7126 like
```
tests/general/TextLintBearTest.py::TextLintBearTest::test_bad_no_start_duplicated_conjunction
  /home/ubuntu/coala-bears/tests/general/TextLintBearTest.py:19: ResourceWarning: unclosed file <_io.TextIOWrapper name='/home/ubuntu/coala-bears/tests/general/textlint_test_files/bad_no_start_duplicated_conjunction.txt' mode='r' encoding='UTF-8'>
    return open(get_testfile_path(name)).readlines()
```
To not rely on ref-counting for object destruction (which isn't anyway an official feature in Python to allow custom destructors), we should close the file manually by using a with-context:
```
with open(get_testfile_path(name)) as fl:
    return fl.readlines()
```

### PR DESCRIPTION
**Pull request by aasiya13**
Fixes #2048 (added by @jayvdb)

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by aasiya13** [comment]
add function to close the text file

File: tests/general/TextLintBearTest.py [hunk 1]
```diff
@@ -14,10 +14,12 @@ def get_testfile_path(name):
                         'textlint_test_files',
                         name)
 
+with open(get_testfile_path(name)) as f1:
+    return f1.readlines()
```
Comment (Makman2) [inline tests/general/TextLintBearTest.py]: this should remain in a function, doesn't work otherwise ;)

**Comment by Makman2** [comment]
Please look at our commit guidelines :+1: :)

File: tests/general/TextLintBearTest.py (hunk 1 above)
Comment (aasiya13) [reply tests/general/TextLintBearTest.py]: ok,I'll make it thanks,

**Comment by Makman2** [comment]
you've committed some unnecessary files, and the code now contains syntax errors :/

**Comment by Makman2** [comment]
If you need any help, feel free to ask on gitter :)

**Comment by newbazz** [comment]
read the http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html for writing commit messages.

**Comment by Makman2** [comment]
Closing due to inactivity.

**Pull request closed without merging**